docs/docs-content/enterprise-version/system-management/ssl-certificate-management.md

```diff
@@ -31,41 +38,107 @@ You can swap out the external endpoint certificate at any time without affecting
 
 - Ensure the certificate is created for the custom domain name you specified for your Palette installation. If you did
   not specify a custom domain name, the certificate must be created for the Palette system console's IP address. You can
-  also specify a load balancer's IP address if you are using a load balancer to access Palette .
+  also specify a load balancer's IP address if you are using a load balancer to access Palette.
 
-## Enablement
+### Enablement
 
-You can upload an SSL certificate in Palette by using the following steps.
+You can update your Palette system address and SSL certificates by using the following steps.
 
 1. Log in to the Palette system console.
 
 2. Navigate to the left **Main Menu** and select **Administration**.
 
-3. Select the tab titled **Certificates**.
+3. Select the tab titled **System Address**.
+
+4. Update your Palette domain in the **System Address (UI and API)** field.
+
+5. Copy and paste the certificate into the **Certificate** field.
 
-4. Copy and paste the certificate into the **Certificate** field.
+6. Copy and paste the certificate key into the **Key** field.
 
-5. Copy and paste the certificate key into the **Key** field.
+7. Copy and paste the certificate authority into the **Certificate Authority** field.
 
-6. Copy and paste the certificate authority into the **Certificate authority** field.
+   ![A view of the certificate upload screen](/palette_system-management_ssl-certificate-management_system-address.webp)
 
-   ![A view of the certificate upload screen](/palette_system-management_ssl-certifiacte-management_certificate-upload.webp)
+8. Click on **Update** to save your changes.
```

Comment:
```suggestion
8. Click **Update** to save your changes.
```

Reply:
Here, I would also direct users to follow up with manually reconciling the address, similar to this:

> 9. If you have any clusters deployed, make sure to reconcile the updated system address for each cluster. Refer to [Reconcile System Address on Deployed Clusters](#reconcile-system...) for more information.